pages/en/lb4/Context.md

```diff
@@ -27,7 +27,8 @@ builtin/in-memory storage mechanism).
 times.
 
 LoopBack's context system allows an unlimited amount of Context instances,
-each of which may have a parent Context.
+each of which may have a parent Context. Each context has a name, which can
```

Comment:
Should there be an example showing users how to name a context?

Reply:
I was considering that but this page is for the concept. If there is a place to show an example, +1.

Reply:
Perhaps in the README for `@loopback/context`?